src/pyOpenMS/pxds/LocalLinearMap.pxd

```diff
@@ -5,15 +5,29 @@ from Matrix cimport *
 cdef extern from "<OpenMS/ANALYSIS/PIP/LocalLinearMap.h>" namespace "OpenMS":
     
     cdef cppclass LocalLinearMap "OpenMS::LocalLinearMap":
+        # wrap-doc:
+            #   Trained Local Linear %Map (LLM) model for peak intensity prediction
+            #   -----
+            #   This class offers a model for predictions of peptide peak heights
+            #   (referred to as intensities) by a Local Linear %Map (LLM) model and
+            #   is the basis of PeakIntensityPredictor.
+            #   -----
+            #   A general introduction to the Peak Intensity Predictor (PIP)
+            #   can be found in the <A HREF="tutorial_pip.html">PIP Tutorial</A>.
```

Comment:
The HREF is probably not working in the code either add a link to the tutorials, or just state PIP Tutorial.